py_client/test_aiaa_server.py

```diff
@@ -90,20 +91,23 @@ def call_server():
 
     client = client_api.AIAAClient(args.server_url)
     session_id = None
+    print('Total tests available: {}'.format(len(tests)))
+    disabled_list = []
     for test in tests:
         name = test.get('name')
         disabled = test.get('disabled', False)
+        disabled = False if args.name and args.name == name else True if args.name else disabled
```

Comment:
Note that `name` could be None here, should we move check at line 111 forward?

The disabled logic is not very clear to me at first sight.
Does it mean the following?

```
if args.name:
    if name and args.name == name:
        disabled = False
    else:
        disabled = True
```